src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hosting/RazorLanguageServerHost.cs

```diff
@@ -83,9 +76,14 @@ private static (JsonRpc, JsonSerializer) CreateJsonRpc(Stream input, Stream outp
         return (jsonRpc, messageFormatter.JsonSerializer);
     }
 
+    public void StartListening()
+    {
+        _jsonRpc.StartListening();
```

Comment:
What was the motivation to move this out of Create, or the constructor?

Reply:
Actually, I intended to put that back! I ran into a test failure that I thought this change would solve, but it turned out to be something else instead. Update incoming.